edx_solutions_api_integration/management/commands/tests/test_rename_api_manager_app.py

```diff
@@ -0,0 +1,85 @@
+"""
+Tests for rename_api_manager_app command
+"""
+import mock
+from datetime import datetime
+
+from south.db import db
+from south.models import MigrationHistory
+from django.db import models, connection
+from django.test import TestCase
+from django.core.management import call_command
+from django.contrib.contenttypes.models import ContentType
+
+
+class RenameApiManagerAppTests(TestCase):
+    """
+    Test suite for renaming api_manager app related database tables
+    """
+    def setUp(self):
+        super(RenameApiManagerAppTests, self).setUp()
+
+        self.old_appname = 'api_manager'
```

Comment:
I think we should have value of old_appname something like `dummy_old_app` so it would not mix/conflict with installed_apps.